packages/@aws-cdk/aws-cloudfront/test/web_distribution.test.ts

```diff
@@ -1186,20 +1186,7 @@ export = {
               customOriginSource: { domainName: 'myorigin.com' },
             }],
           });
-        }, 'connectionTimeout: You can specify a number of seconds between 1 and 10 (inclusive).');
-        test.done();
-      },
-      'connectionTimeout = -1'(test: Test) {
-        const stack = new cdk.Stack();
-        test.throws(() => {
-          new CloudFrontWebDistribution(stack, 'Distribution', {
-            originConfigs: [{
-              behaviors: [{ isDefaultBehavior: true }],
-              connectionTimeout: cdk.Duration.seconds(-1),
```

Comment:
Note: Duration.seconds(-1) throws an exception, making this test fail for the wrong reason. Deleted it as unnecessary.